webplugin/js/app/kommunicate-ui.js

```diff
@@ -500,12 +500,11 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
     }
     },
     displayPopupChatTemplate: function(popupChatContent, chatWidget, mckChatPopupNotificationTone) {
-
-        var isPopupEnabled = kommunicateCommons.isObject(chatWidget) && chatWidget.popup;
+        var disableGreetingMessage = kommunicateCommons.isObject(chatWidget) && chatWidget.hasOwnProperty('disableGreetingMessageInMobile') ? chatWidget.disableGreetingMessageInMobile : false;
```

Comment:
can you inert this, anything with disable: true/false is confusing to understand

Reply:
done